client.js

```diff
@@ -142,9 +142,6 @@ DHT.prototype.put = function (opts, cb) {
   if (isMutable && opts.cas !== undefined && typeof opts.cas !== 'number') {
     throw new Error('opts.cas must be an integer if provided')
   }
-  if (isMutable && !opts.k) {
-    throw new Error('opts.k ed25519 public key required for mutable put')
-  }
```

Comment:
why was this check removed?

Reply:
I think because `var isMutable = !!opts.k` ?